widgets/social-media-buttons/tpl/default.php

```diff
@@ -6,7 +6,19 @@
 		if( !empty($instance['design']['hover']) ) $classes[] = 'ow-button-hover';
 		$classes[] = "sow-social-media-button-" . sanitize_html_class( $network['name'] );
 		$classes[] = "sow-social-media-button";
-		$title = empty( $network['icon_title'] ) ? sprintf( __( '%s on %s', 'so-widgets-bundle' ), get_bloginfo( 'name' ), ucwords( str_replace( '-', ' ', $network['name'] ) ) ) : $network['icon_title'];
+
+		if( ! empty( $network['icon_title'] ) ) {
+			$title = $network['icon_title'];
+		} else {
+			if ( $network['name'] == 'email' || $network['name'] == 'phone' ) {
+				if ( $network['title'] == 'envelope' ) $network['name'] == 'email';
```

Comment:
This is quite confusing. Where does the `$network['title']` come from? And then it's just doing a check to see if `$network['name']` is equal to 'email' without assigning it to anything?

Reply:
That actually contains two major typos. 😞 
It's not email, it's Envelope. It's also not `$network['title']`, it's `$network['name']`. I had thought I fixed that before committing it but I guess I didn't - sorry about that!